android/quest/src/main/res/values/colors.xml

```diff
@@ -10,6 +10,9 @@
     <color name="dodger_blue">#1976D2</color>
     <color name="gray">#FF808080</color>
     <color name="light_gray">#FFD3D3D3</color>
+    <color name="other_light_gray">#F7F7F7</color>
+    <color name="dark_gray">#6F7274</color>
```

Comment:
You can instead use `DefaultColor` from `Colors.kt` and change the transparency e.g.

```kotlin 
DefaultColor.copy(alpha = 0.5) // Moderately transparent
DefaultColor.copy(alpha = 0.2) // Highly transparent
```

No need for defining same color code separately with different shades.

Reply:
thanks @ellykits  for this practice.